var/spack/repos/builtin/packages/py-wradlib/package.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyWradlib(PythonPackage):
+    """wradlib is designed to assist you in the most important steps of
+    processing weather radar data. These may include: reading common data
+    formats, georeferencing, converting reflectivity to rainfall intensity,
+    identifying and correcting typical error sources (such as clutter or
+    attenuation) and visualising the data."""
+
+    homepage = "https://docs.wradlib.org"
+    url      = "https://pypi.io/packages/source/w/wradlib/wradlib-1.5.0.tar.gz"
+
+    version('1.5.0', sha256='9bf0742d7235ea830e83c2269f6b5d1afd83d92696efce0a7bcdb0c4f6604784')
+
+    depends_on('py-setuptools', type='build')
+
+    # https://docs.wradlib.org/en/stable/index.html
+    conflicts('^python@:2.999', when='@1.3:')
+
+    # recommended versions from https://docs.wradlib.org/en/stable/installation.html#dependencies
+    depends_on('py-numpy@1.16:', type=('build', 'run'))
+    depends_on('py-matplotlib@3.0.2:', type=('build', 'run'))
+    depends_on('py-scipy@1.2.0:', type=('build', 'run'))
+    depends_on('py-h5py@2.9:', type=('build', 'run'))
+    depends_on('py-netcdf4@1.4.2:', type=('build', 'run'))
+    depends_on('py-xarray@0.11.3:', type=('build', 'run'))
+    depends_on('py-xmltodict@0.11:', type=('build', 'run'))
+    depends_on('py-pygdal@2.4.0:', type=('build', 'run'))
```

Comment:
Not sure if it's better to use `py-pygdal` or `gdal+python` for this one. I maintain the `gdal` package in Spack, but I've never used the `py-pygdal` package before. There's also a `py-gdal` package (not yet in Spack, but it's available in PyPI) that we could use.

Reply:
Yes, PyPI only has `pygdal` (`py-pygdal` in Spack).